### PR TITLE
react/quiz

### PR DESCRIPTION
**Pull request by lontsimesmer**
This project is all about react hooks and context

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -21,6 +21,7 @@
       {
         "extensions": [".js", ".jsx"]
       }
-    ]
+    ],
+    "prettier/prettier":["error", {"endOfLine": "auto"}]
```
Comment (gilons) [inline .eslintrc:25]: Why is this setup needed?

File: README.md [hunk 2]
```diff
@@ -1,70 +1,30 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+# react-quiz-app
```
Comment (gilons) [inline README.md:1]: Nice adding the readme here.

File: src/Components/Hooks/useFetch.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { useEffect, useState } from 'react';
+import fetchQuestions from '../../Api';
+
+export default function useFetch() {
```
Comment (gilons) [inline src/Components/Hooks/useFetch.jsx:4]: nice approach. But the name `useFetch` is too vague to be tied to a single HTTP method and endpoint. I will suggest something like `useQuestions`

File: src/Components/Hooks/useIncrement.jsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { useState } from 'react';
+
+export default function useIncrement(defaultValue = null) {
```
Comment (gilons) [inline src/Components/Hooks/useIncrement.jsx:3]: Nice hooks usage

File: src/Pages/Home/Home.jsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
+import { useContext } from 'react';
+import { Link } from 'react-router-dom';
+import { QuestionContext } from '../../Context';
+import Styles from './Home.module.css';
```
Comment (gilons) [inline src/Pages/Home/Home.jsx:4]: No need for the name to be capitalized.

File: src/Pages/Result/Results.jsx [hunk 6]
```diff
@@ -0,0 +1,53 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Styles from './Results.module.css';
+import { QuestionConsumer } from '../../Context';
+
+export default function Result() {
+  return (
+    <QuestionConsumer>
```
Comment (gilons) [inline src/Pages/Result/Results.jsx:9]: use hooks instead

File: src/Pages/Page.jsx [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Home from './Home/Home';
+import QuizPage from './Quiz/QuizPage';
+import Result from './Result/Results';
+
+export default function Page() {
+  return (
+    <div>
+      <Home />
```
Comment (gilons) [inline src/Pages/Page.jsx:9]: Why are you rendering these components at the same time? I guess it's not used, please remove it.

**Review by gilons — APPROVED** [review]

File: .eslintrc (hunk 1 above)
Comment (lontsimesmer) [reply .eslintrc:25]: For prettier code sir

File: README.md (hunk 2 above)
Comment (lontsimesmer) [reply README.md:1]: Thank you sir

File: src/Components/Hooks/useFetch.jsx (hunk 3 above)
Comment (lontsimesmer) [reply src/Components/Hooks/useFetch.jsx:4]: Ok sir i consider

File: src/Pages/Result/Results.jsx (hunk 6 above)
Comment (lontsimesmer) [reply src/Pages/Result/Results.jsx:9]: This was just to understand the use of react context

File: src/Pages/Page.jsx (hunk 7 above)
Comment (lontsimesmer) [reply src/Pages/Page.jsx:9]: Understood sir

**Pull request merged**